### PR TITLE
Set moving load 17

### PR DESCRIPTION
**Pull request by morettid**

### CODE REVIEW

**Comment by morettid** [comment]
Not yet ready for review. It will follow after Point Load (issue #16 and pull request #30).

File: stem/__init__.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+
+DOMAIN = "PorousDomain"
```
Comment (aronnoordam) [inline stem/__init__.py:2]: I would put this in kratos_IO.py, as it is really Kratos specific

File: stem/load.py [hunk 2]
```diff
@@ -25,6 +26,28 @@ class PointLoad(LoadParametersABC):
...
+    Class containing the load parameters for a moving load.
+
+    Attributes:
+        load (Union[List[float], List[str]]): Entity of the load [N] in the 3
+             directions. Can be defined as strings (when function of time) or as float.
+             Mixed types are not accepted.
+        direction (List[float]):  Direction of the moving load.
+        velocity (Union[float, str]): Velocity of the moving load [m/s].
+        origin (List[float]): Starting coordinates of the moving load [m].
+        offset (float): Offset of the moving load.
```
Comment (aronnoordam) [inline stem/load.py:41]:
```suggestion
        offset (float): Offset of the moving load [m].
```

File: stem/load.py [hunk 3]
```diff
@@ -25,6 +26,28 @@ class PointLoad(LoadParametersABC):
...
+@dataclass
+class MovingLoad(LoadParametersABC):
+    """
+    Class containing the load parameters for a moving load.
+
+    Attributes:
+        load (Union[List[float], List[str]]): Entity of the load [N] in the 3
+             directions. Can be defined as strings (when function of time) or as float.
+             Mixed types are not accepted.
+        direction (List[float]):  Direction of the moving load.
```
Comment (aronnoordam) [inline stem/load.py:38]:
```suggestion
        direction (List[int]):  Direction of the moving load (-1 or +1 in x, y, z direction) [-].
```

File: stem/load.py [hunk 4]
```diff
@@ -25,6 +26,28 @@ class PointLoad(LoadParametersABC):
...
+        velocity (Union[float, str]): Velocity of the moving load [m/s].
+        origin (List[float]): Starting coordinates of the moving load [m].
+        offset (float): Offset of the moving load.
+    """
+
+    load: Union[List[float], List[str]] = field(default_factory=lambda: [0.0, 0.0, 0.0])
+    direction: List[float] = field(default_factory=lambda: [1.0, 1.0, 1.0])
```
Comment (aronnoordam) [inline stem/load.py:45]:
```suggestion
    direction: List[float] = field(default_factory=lambda: [1, 1, 1])
```

File: tests/test_Kratos_IO.py [hunk 5]
```diff
@@ -1,41 +1,47 @@
...
     def test_create_load_process_dictionary(self):
         """
         Test the creation of the load process dictionary for the
         ProjectParameters.json file
         """
         # define load(s) parameters
-        point_load_parameters = PointLoad(active=[True, True, True],
-                                          value=[1000, 0, 0] )
+        # point load
+        point_load_parameters = PointLoad(active=[True, True, True], value=[1000, 0, 0])
```
Comment (aronnoordam) [inline tests/test_Kratos_IO.py:16]: I think its better to set "active" to non-default values, same for the moving load parameters

Comment (morettid) [reply tests/test_Kratos_IO.py:16]: Agreed

**Review by aronnoordam — APPROVED** [review]

**Pull request merged**